### ISSUE TITLE
Map matcher

### ISSUE THREAD

**Issue by luigidalmare** [objecttrouve/convenience-matchers#1]
hi,

i wonder if there is a matcher for map type. something like 

```
mapOf(String.class, String.class).withEntries("key", "value", "key", "value",...)
```

it would be very useful

**Comment by objecttrouve** [comment]
Sounds like a reasonable idea. 
Let me think about it. It's a long time since I last touched this...

**Comment by objecttrouve** [comment]
Fixed in [1.2.0](https://github.com/objecttrouve/convenience-matchers/releases/tag/1.2.0). 

Thanks for the contribution, @luigidalmare!

**Comment by objecttrouve** [comment]
Nothing left to do here.
